dojo_plugin/utils/stats.py

```diff
@@ -34,24 +34,10 @@ def get_dojo_stats(dojo):
     chart_solves = []
     chart_users = []
 
-    return {
-        'users': total_users,
-        'challenges': total_challenges,
-        'visible_challenges': visible_challenges,
-        'solves': total_solves,
-        'recent_solves': [],
-        'trends': {
-            'solves': 0,
-            'users': 0,
-            'active': 0,
-            'challenges': 0,
-        },
-        'chart_data': {
-            'labels': chart_labels,
-            'solves': chart_solves,
-            'users': chart_users
-        }
-    }
```

Comment:
We actually intentionally have this disabled right now for performance reasons. We need a smarter solution where the background tasks update recent results, and api requests just always fetch from that. As our caching system currently exists, we've seen production melt down several times.